requirements.txt

```diff
@@ -24,3 +24,4 @@ retrying!=1.3.0,>=1.2.3 # Apache-2.0
 stevedore>=1.20.0 # Apache-2.0
 grpcio>=1.12.0 # Apache-2.0
 protobuf>=3.6.0 # 3-Clause BSD
+prometheus_client>=0.7.1 # Apache-2.0
```

Comment:
We might have troubles providing this one, I'll check if it's available somewhere.

Reply:
Thanks! An older version works as well (prometheus_client>=0.6.0 )

Reply:
Seems like in repos we only have 0.6.0 version of that lib, which is maximum we can use in 4.5.